src/components/inputWithAutocomplete.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+class InputWithAutocomplete extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isVisibleHints: false
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.fillInputWithCheckedHint = this.fillInputWithCheckedHint.bind(this);
+        this.closeHints = this.closeHints.bind(this);
+        this.findHints = this.findHints.bind(this);
```

Comment:
findHints is not passed as a callback to some other component. Why did you bind it?